r_gl_fbo.cpp

```diff
@@ -0,0 +1,63 @@
+#include "r_gl_fbo.h"
+
+#include <SDL.h>
+#include <glad/glad.h>
+#include <stdio.h>
+
+#include "r_gl_rendertexture.h"
+
+// Not usable in this state! Call any of the non-default Ctors to
+// get the FBO into valid state.
+CglFBO::CglFBO() { 
+    m_Width = 0;
+    m_Height = 0;
+    m_hFBO = 9999; // TODO: (Michael): Can the handle be 0? Check GL docs!
+}
```

Comment:
Sometimes it can be useful to have object in uninitialized state. C++'s RAII is actually a dumb idea. But I agree that we could just delete it and set those values in the header file. Not sure...